sync/class.jetpack-sync-defaults.php

```diff
@@ -271,6 +271,7 @@ static function is_whitelisted_option( $option ) {
 	static $default_sync_wait_threshold = 5; // only wait before next send if the current send took more than X seconds
 	static $default_max_queue_size = 1000;
 	static $default_max_queue_lag = 900; // 15 minutes
+	static $default_max_dequeue_time = 15; // 15 seconds
```

Comment:
As discussed, I added this change and tweaked it a bit (since defining `static` vars at runtime is not allowed in PHP, and also because this value could be _highly_ dynamic within the context of a single application, between web vs cron vs shell vs other environments.